app/feature.go

```diff
@@ -100,14 +100,6 @@ func NewFeatureCommand() (CommandOut, error) {
 			Usage:   "feature commands",
 			Hidden:  true,
 			Subcommands: []*cli.Command{
-				{
-					Name:    "toggle-export",
-					Aliases: []string{"te"},
-					Usage:   "switch export on/off",
```

Comment:
can you remove export reference from here too? https://github.com/temporalio/tcld/blob/6196555f9a9d1c5aab65880858efe9700a554124/app/feature.go#L14